java/src/jmri/jmrix/marklin/cdb/serialdriver/Bundle.java

```diff
@@ -0,0 +1,93 @@
+package jmri.jmrix.marklin.cdb.serialdriver;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.Locale;
+import javax.annotation.CheckReturnValue;
+import javax.annotation.CheckForNull;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+@ParametersAreNonnullByDefault
+@CheckReturnValue
+@SuppressFBWarnings(value = "NM_SAME_SIMPLE_NAME_AS_SUPERCLASS",justification="Desired pattern is repeated class names with package-level access to members")
+
+@javax.annotation.concurrent.Immutable
+
+/**
+ * Provides standard access for resource bundles in a package.
+ * 
+ * Convention is to provide a subclass of this name
+ * in each package, working off the local resource bundle name.
+ *
+ * @author      Bob Jacobsen  Copyright (C) 2012
+ * @since       3.7.2
+ */
+public class Bundle extends jmri.jmrix.tams.Bundle {
```

Comment:
This class should inherit the Bundle class of the super package which in this case is `jmri.jmrix.marklin.cdb`. If that package doesn't have a Bundle class, you can add it and let that Bundle class inherit the Bundle class of `jmri.jmrix.marklin`.